PrecisionGazeMouse/MouseController.cs

```diff
@@ -7,17 +7,19 @@
 
 namespace PrecisionGazeMouse
 {
+    public delegate void UpdateCursorPosition(Point cursorPosition);
+
     class MouseController
     {
         private static readonly log4net.ILog log = log4net.LogManager.GetLogger(System.Reflection.MethodBase.GetCurrentMethod().DeclaringType);
 
-        WarpPointer warp;
-        PrecisionPointer prec;
+        WarpPointer warp = new NoWarpPointer();
+        PrecisionPointer prec = new NoPrecisionPointer();
```

Comment:
These changes ended up being unnecessary, but I kept them because I think it's a good practice to avoid uninitialized states wherever possible.